fontir/src/ir.rs

```diff
@@ -543,6 +548,8 @@ impl GlobalMetrics {
         self.values_mut(metric).insert(pos, value.into());
     }
 
+    /// Specify the value of a metric at a specific location, but only if there
+    /// is not one already provided. Returns the winning value.
```

Comment:
Treat docs like git commits: one short summary line, then an empty line, then any elaboration as required, e.g.:

```suggestion
    /// Set the value of a metric at a specific location, if none exists.
    ///
    /// If a value already exists at this location, it is not changed.
    /// 
    /// Returns the (potentially new, potentially preexisting) value.
```